scenariolib/event_click_test.go

```diff
@@ -0,0 +1,46 @@
+package scenariolib_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/coveo/uabot/scenariolib"
+)
+
+func TestClickEventValid(t *testing.T) {
+	var testEventJson = []byte(`{"probability": 1, "docNo": -1, "offset": 2, "quickview": false, "customData": {"data1": "one"}}`)
+	event := &scenariolib.ClickEvent{}
+
+	// Test unmarshal json.
+	if err := json.Unmarshal(testEventJson, event); err != nil {
+		t.Errorf("Error Unmarshaling ClickEvent: %s", err)
```

Comment:
There's no "assert" in Go ? You need to `t.ErrorF` to fail the test ?

Reply:
Nope there are third party libs for assert, but otherwise theres no core assert.

Reply:
Should you have a test to make sure invalid events are invalid? like when getting something like `"offset":-1` or `"offset":false` (happens often with copy-paste)